ResearchKit/Common/ORKHelpers.m

```diff
@@ -251,6 +251,12 @@ void ORKEnableAutoLayoutForViews(NSArray *views) {
     return bundle;
 }
 
+NSBundle *ORKDefaultLocaleBundle() {
+    NSString * path = [ORKBundle() pathForResource:@"en" ofType:@"lproj"];
+    NSBundle * bundle = [NSBundle bundleWithPath:path];
```

Comment:
Wouldn't it be nicer to get the default localization (`en`) by using `[[NSBundle mainBundle] objectForInfoDictionaryKey:@"CFBundleDevelopmentRegion"]` rather than hardcoding it?

Also, remove spaces between `*` and variable name.